Detectors/EMCAL/reconstruction/include/EMCALReconstruction/CTFCoder.h

```diff
@@ -188,7 +195,14 @@ o2::ctf::CTFIOSize CTFCoder::decode(const CTF::base& ec, VTRG& trigVec, VCELL& c
     firstEntry = cellVec.size();
 
     for (uint16_t ic = 0; ic < entries[itrig]; ic++) {
-      cell.setPacked(tower[cellCount], cellTime[cellCount], energy[cellCount], status[cellCount]);
+      /*
+      uint16_t packedEnergy = energy[cellCount];
+      if (isVersion1) {
+        packedEnergy = o2::emcal::Cell::V0toV1(energy[cellCount], status[cellCount]);
+      }
+      cell.initialiseFromEncoded(tower[cellCount], cellTime[cellCount], packedEnergy, status[cellCount]);
+      */
+      cell.initialiseFromEncoded(tower[cellCount], cellTime[cellCount], energy[cellCount], status[cellCount]);
       cellVec.emplace_back(cell);
```

Comment:
use directly `cellVec.emplace_back(tower[cellCount], cellTime[cellCount], packedEnergy, status[cellCount]);`

Reply:
No, emplace_back calls the constructor of the same signature, just the construction happens in place

Reply:
But this will then need a dedicated constructor. OK, can in principle be added.